src/settings.js

```diff
@@ -19,6 +19,10 @@ export function updateInstance(cozy, instance) {
   return cozyFetchJSON(cozy, 'PUT', `/settings/instance`, instance)
 }
 
+export function getCapabilities(cozy) {
+  return cozyFetchJSON(cozy, 'GET', `/settings/capabilities`)
+}
```

Comment:
I think that we should add features to cozy-client instead of cozy-client-js. Instantiating a cozy-client from a cozy-client-js client is fairly easy and would pave the way for the complete migration to cozy-client.

Reply:
I am uncomfortable to have this here where cozy-client does not have this feature.

Reply:
https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/CozyClient.js#L185

Reply:
We have it in cozy-client! SettingCollection.get('capabilities')

Reply:
>Instantiating a cozy-client from a cozy-client-js client is fairly easy and would pave the way for the complete migration to cozy-client.

I told that to @taratatach a few weeks ago. We should really push in that direction.